postgresql_proxy/constants.py

```diff
@@ -0,0 +1,42 @@
+
+ALLOWED_CONNECTION_PARAMETERS = [
```

Comment:
We probably have to adjust this list for each new postgresql version, but seems like a good solution for now!